app/styles/analysis.scss

```diff
@@ -77,3 +77,6 @@
 .table--analysis tr.selected td {
   border-color: lighten($color-primary, 20%) !important;
 }
+.blue-checked {
+  color: darken($color-primary, 10);
```

Comment:
Still not in the [new sass module syntax style](https://sass-lang.com/documentation/modules/color#darken)

See example [here](https://github.com/projectcaluma/ember-caluma/blob/ce0833f6f308536e95b6ebbd2516b526660d87c5/packages/form/app/styles/_flatpickr.scss#L24)